components/analysis/component.jsx

```diff
@@ -80,31 +78,33 @@ class AnalysisComponent extends PureComponent {
           {loading && (
             <Loader className={cx('analysis-loader', { fetching: loading })} />
           )}
-          {location.type && location.adm0 && (loading || (!loading && error)) && (
-            <div className={cx('cancel-analysis', { fetching: loading })}>
-              {!loading && error && !geostoreError && (
+          {location.type &&
+            location.adm0 &&
+            (loading || (!loading && error)) && (
+              <div className={cx('cancel-analysis', { fetching: loading })}>
+                {!loading && error && !geostoreError && (
+                  <Button
+                    className="refresh-analysis-btn"
+                    onClick={() => handleFetchAnalysis(endpoints)}
+                  >
+                    REFRESH ANALYSIS
+                  </Button>
+                )}
                 <Button
-                  className="refresh-analysis-btn"
-                  onClick={() => handleFetchAnalysis(endpoints)}
+                  className="cancel-analysis-btn"
+                  onClick={handleCancelAnalysis}
                 >
-                  REFRESH ANALYSIS
```

Comment:
is this requested in a specific tickret? changing "REFRESH" to "CANCEL", seems it isn't part of the node update ticket 🤔

Reply:
Nops, it was a trick. If you compare the code nothing changed. It was just the ESLint. I've just checked it this morning to make sure haha